packages/labs/ssr/src/lib/dom-shim.ts

```diff
@@ -15,12 +15,17 @@
 import fetch from 'node-fetch';
 
 /**
- * Constructs a fresh instance of the "window" vm context to use for
- * evaluating user SSR code.  Includes a minimal shim of DOM APIs.
+ * Constructs a fresh instance of the "window" vm context to use for evaluating
+ * user SSR code. Includes a minimal shim of DOM APIs.
  *
+ * @param includeJSBuiltIns Whether certain standard JS context globals like
+ *  `console` and `setTimeout` should also be added to the global. Should
+ *  generally only be true when adding window to a sandboxed VM context that
+ *  starts with nothing.
  * @param props Additional properties to add to the window global
  */
 export const getWindow = (
+  includeJSBuiltIns = false,
```

Comment:
optional booleans are hard to read at callsites. Make an options bag?

Reply:
Yeah, better. Done.